softwerkskammer/lib/mailsender/mailsenderService.js

```diff
@@ -140,11 +185,10 @@ module.exports = {
     }
   },
 
-  sendMailToAllMembers: async function sendMailToAllMembers(message) {
+  sendMailToAllMembers: async function sendMailToAllMembers(message, sender) {
```

Comment:
nochmal die frage nach dem sender, der ist doch in der message schon drin?

Reply:
Nein, nur der Name und die Mailadresse, daraus wieder einen Member zu bauen, fand ich etwas doof bzw. war mir auch nicht sicher, ob das dann immer so klappt.
Ich wollte auch die `Message` nicht um eine weiteres Feld erweitern (eben den `sender`), aber wenn das ok ist, kann ich das auch machen.

Reply:
ja, bitte

Reply:
Wozu brauchst Du den member?

Reply:
Ich habe mal geschaut: prinzipiell würde gehen, wenn man den Sender aus der `senderAddress` in der `message` rekonstruiert. Dann gibt `displayName()` aber `"null null"` zurück.
Daher wäre mein Vorschlag, doch `sender` in `Message` als Property bereitzustellen.
Das habe ich auch im letzten Commit von gerade eben so umgesetzt.